.circleci/config.yml

```diff
@@ -111,6 +100,9 @@ jobs:
       - run:
           name: Fetch back the produced OVA image
           command: rsync -avz -e 'ssh -o StrictHostKeyChecking=no' root@metal:/home/ova/builds .
+      - run:
+          name: Remove unnecessary files before persisting and deploying
+          command: rm builds/README.md builds/*.box
```

Comment:
CircleCI artifacts should contain both `.ova` and `.box` files, otherwise I won't be able to deploy the Vagrant box to Vagrant Cloud in #10.
While Github releases needs only `.ova`.

So this way we ship :
- `.box` -> Vagrant Cloud
- `.ova` -> Github Releases